Assets/GameBrowserHandler.cs

```diff
@@ -28,4 +39,55 @@ void RefreshMatches()
             newButton.GetComponent<GameBrowserEntry>().SetMatchDetails(match);
         }
     }
+
+    void Update() {
+
+        if (control is null)
```

Comment:
This implementation could be improved:
if ((Input.GetKey(KeyCode.LeftControl) || Input.Get(KeyCode.RightControl)) && Input.GetKeyUp(KeyCode.G)) {
  // Ctrl-G pressed
}

Even if that was unavailable, code like this belong in a helper class that tacks keys individually rather than having a shared flag for two keys (which inevitably leads to situations where tracking fails).

At this point, there is no strict need for ctrl-g specifically, so if the GetKey() method doesn't work, just skip the ctrl modifier and use a raw "G" press instead.

Reply:
Changed to a simple G press and the input field now accepts integers only.